README.md

```diff
@@ -1,64 +1,4 @@
-# mlx_steering_dump_parser
-This is Mellanox SW steering parser and triggering for dump files in **CSV format**.
-The supported dump files are those generated by ConnectX5 and ConnectX6DX.
+This is the HW/SW steering dump parsing tools.
```

Comment:
need to rebase - readme file has some updates

Reply:
Dropped last patch that touch the SWS directories.
will merge this series then will move SWS files to another directory.

Reply:
done